xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererDRMPRIME.h

```diff
@@ -12,11 +12,35 @@
 #include "cores/VideoPlayer/VideoRenderers/BaseRenderer.h"
 #include "windowing/gbm/WinSystemGbmEGLContext.h"
 
+class CVideoLayerBridgeDRMPRIME
+  : public CVideoLayerBridge
+{
+public:
+  CVideoLayerBridgeDRMPRIME(std::shared_ptr<CDRMUtils> drm);
+  ~CVideoLayerBridgeDRMPRIME();
+  void Disable() override;
+
+  virtual void Configure(CVideoBufferDRMPRIME* buffer);
+  virtual void SetVideoPlane(CVideoBufferDRMPRIME* buffer, const CRect& destRect);
```

Comment:
do these need to be virtual?

Reply:
`Configure` is `override` in my Rockchip HDR work-in-progress class, I made Configure/SetVideoPlane virtual to minimize the amount of changes needed for Rockchip HDR

Reply:
I'd rather have all relevant changes in the specific PR but that's just me. Not really a big deal